### DISCUSSION TITLE
Could NOT find Boost

### DISCUSSION THREAD

**Discussion by wcc19840827** [android/ndk#1678] [Q&A]
Boost not found when I use android.toolchain.cmake

Use `find_package(Boost 1.77.0 COMPONENTS system thread regex ${ARGS})` to find boost lib

 the error is `-- Could NOT find Boost (missing: Boost_DIR)`

**Comment by DanAlbert** [comment]
How did you install boost for Android? Does whatever distribution method you used _expect_ this to work?

**Comment by lyfadvance** [comment]
I found that the reason for this issue is that the 'find_package' command recursively searches for the '. cmake' file under the Linux gcc toolchain, while under the Android ndk toolchain, it does not recursively search for the '. cmake' file。
Have you sovled this problem?
